<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>distMeasure</class>
 <widget class="QWidget" name="distMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>484</width>
     <height>274</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <item>
     <widget class="QWidget" name="wt_liveView" native="true">
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>272</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>272</width>
        <height>16777215</height>
       </size>
      </property>
      <widget class="QLabel" name="lb_zoomVal_2">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>208</y>
         <width>359</width>
         <height>59</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;PRESS ENTER TO ACCEPT OR&lt;/p&gt;&lt;p&gt;KEYPAD TO ENTER DISTANCE MANUALLY&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_range">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>50</y>
         <width>270</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_top">
       <property name="geometry">
        <rect>
         <x>78</x>
         <y>10</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>6</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="laserRec">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>110</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ftc1">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>90</y>
         <width>270</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ftc2">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>130</y>
         <width>270</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="lb_ftc3">
       <property name="geometry">
        <rect>
         <x>66</x>
         <y>170</y>
         <width>270</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Abyssinica SIL</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="fr_panel">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>270</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>100</width>
        <height>270</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/hh1Graphics/blue-bkg);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pb_exit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/hh1Graphics/blue-button);</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_enter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/hh1Graphics/blue-button);</string>
         </property>
         <property name="text">
          <string>ENTER</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="vkILine" name="le_keypad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>66</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/hh1Graphics/blue-button);</string>
         </property>
         <property name="text">
          <string>KEYPAD</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_value">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>66</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>66</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/hh1Graphics/blue-button);</string>
         </property>
         <property name="text">
          <string>REDO</string>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vkILine</class>
   <extends>QLineEdit</extends>
   <header>vkiline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
